src/App.js

```diff
@@ -5,12 +5,13 @@ import LeftColumn from './modules/Left-column';
 import Carousel from './modules/Carousel';
 import ProductBox from './modules/ProductBox';
 import Foot from './modules/Footer';
-{/* <ProductBox product={ props.products.products[ind] } */ }
+
+
 const App = (props) => {
-  const test =
+  const items =
     props.products.products.map(
       (cur, ind) => {
-        return <ProductBox product={props.products.products[ind]} />
+        return <ProductBox product={props.products.products[ind]} key={ind} />
```

Comment:
this is fine, but make sure you understand that cur is the same as props.products.products[ind].
less typing